judge/fixtures/language_small.json

```
@@ -263,7 +263,7 @@
          "name":"Brain****",
          "short_name":null,
          "common_name":"Brain****",
-         "ace":"bf",
+         "ace":"plaintext",
```

```suggestion
         "ace":"plain_text",
```

Looks like the name should actually be `plain_text`.

---

Yep that looks more correct, thanks for pointing it out 😅